axelor-production/src/main/resources/views/BillOfMaterial.xml

```diff
@@ -20,15 +20,15 @@
           onLoad="action-bill-of-material-attrs-price-and-qty-digits,action-bill-of-material-record-count-number-of-components">
 
 		<menubar>
-			<menu title="Reports">
-				<item title="Print" action="action-bill-of-material-method-print"/>				
+			<menu name="billOfMaterialReportsMenu" title="Reports">
+				<item name="printItem" title="Print" action="action-bill-of-material-method-print"/>				
 			</menu>
 
-            <menu title="Options">
-                <item title="Generate new version"
+            <menu name="billOfMaterialOptionsMenu" title="Options">
+                <item name="generateNewVersionItem" title="Generate new version"
                     action="action-bill-of-material-check-other-version,action-bill-of-material-generate-new-version"
                     if="__config__.app.getApp('production').manageBillOfMaterialVersion"/>
-                <item title="Define as default bill of material of product"
+                <item name="defineDefaultBillOfMaterialOfProductItem" title="Define as default bill of material of product"
```

Comment:
don't give too long names, make it short over here as 'defineDefaultBOMOfProductItem'